Mathlib/CategoryTheory/EssentialImage.lean

```diff
@@ -168,4 +168,10 @@ instance Full.toEssImage (F : C ⥤ D) [Full F] : Full F.toEssImage :=
   Full.ofCompFaithful F.toEssImage F.essImageInclusion
 #align category_theory.full.to_ess_image CategoryTheory.Full.toEssImage
 
+instance instEssSurjId : EssSurj (𝟭 C) where
+  mem_essImage Y := ⟨Y, ⟨Iso.refl _⟩⟩
+
+theorem EssSurj.ofNatIso {F G : C ⥤ D} [EssSurj F] (α : F ≅ G) : EssSurj G where
```

Comment:
Could we rename this to
```suggestion
theorem NatIso.map_EssSurj {F G : C ⥤ D} [EssSurj F] (α : F ≅ G) : EssSurj G where
```
to enable dot-notation? Feel free to improve on my name suggestion.

Reply:
Unfortunately this wouldn't help dot notation: it would have to be in the `Iso` namespace, but I guess that works.

Reply:
```suggestion
theorem Iso.map_essSurj {F G : C ⥤ D} [EssSurj F] (α : F ≅ G) : EssSurj G where
```